src/extester.ts

```diff
@@ -48,13 +42,6 @@ export class ExTester {
     constructor(storageFolder: string = DEFAULT_STORAGE_FOLDER, releaseType: ReleaseQuality = ReleaseQuality.Stable, extensionsDir?: string) {
         this.code = new CodeUtil(storageFolder, releaseType, extensionsDir);
         this.chrome = new DriverUtil(storageFolder);
-
-        if (process.versions.node > NODEJS_VERSION_MAX) {
```

Comment:
I would still rather have some notification in console for users to let them they should not use later then 18.x.x

or that 18.x.x+ versions are not tested and supported